ui2/src/pages/clearing/Members.tsx

```diff
@@ -0,0 +1,123 @@
+import { KeyboardArrowRight } from '@material-ui/icons';
+import { Service } from '@daml.js/da-marketplace/lib/Marketplace/Clearing/Service';
+import { MemberStanding } from '@daml.js/da-marketplace/lib/Marketplace/Clearing/Model';
+import { ServicePageProps } from '../common';
+import { Header } from 'semantic-ui-react';
+import Tile from '../../components/Tile/Tile';
+import StripedTable from '../../components/Table/StripedTable';
+import MarginCallModal from './MarginCallModal';
+import MTMCalculationModal from './MTMCalculationModal';
+
+const ClearingMembersComponent: React.FC<RouteComponentProps & ServicePageProps<Service>> = ({
+  history,
+  services,
+}: RouteComponentProps & ServicePageProps<Service>) => {
+  const party = useParty();
+
+  const accounts = useStreamQueries(AssetSettlementRule).contracts;
+  const deposits = useStreamQueries(AssetDeposit).contracts;
+  const standings = useStreamQueries(MemberStanding).contracts;
+  const ccpDeposits = deposits.filter(
+    d => d.payload.account.id.label === services[0].payload.ccpAccount.id.label
+  );
+
+  const formatter = new Intl.NumberFormat('en-US', {
+    style: 'currency',
+    currency: 'USD',
+  });
+
+  return (
+    <div className="assets">
+      <Tile header={<h2>Actions</h2>}>
+        <MarginCallModal services={services} />
+        <MTMCalculationModal services={services} />
+      </Tile>
+      <Header as="h2">Holdings</Header>
+      <StripedTable
+        headings={['Member', 'Clearing Account', 'Margin Account', 'In Good Standing', 'Details']}
+        rows={services.map(s => {
+          const standing = standings.find(
+            standing => standing.payload.customer === s.payload.customer
+          );
+          const clearingDeposits = deposits.filter(
+            d => d.payload.account.id.label == s.payload.clearingAccount.id.label
+          );
+          const marginDeposits = deposits.filter(
+            d => d.payload.account.id.label == s.payload.marginAccount.id.label
+          );
+          const clearingAmount = clearingDeposits.reduce(
+            (acc, val) => acc + Number(val.payload.asset.quantity),
+            0
+          );
+          const marginAmount = marginDeposits.reduce(
+            (acc, val) => acc + Number(val.payload.asset.quantity),
+            0
+          );
+          const standingText =
+            !!standing && standing.payload.marginSatisfied && standing.payload.mtmSatisfied
+              ? 'Yes'
+              : 'No';
+          return [
+            <>{s.payload.customer}</>,
+            formatter.format(clearingAmount),
+            formatter.format(marginAmount),
+            standingText,
+            <IconButton
+              color="primary"
+              size="small"
+              component="span"
+              onClick={() => history.push('/app/clearing/member/' + s.contractId.replace('#', '_'))}
+            >
+              <KeyboardArrowRight fontSize="small" />
+            </IconButton>,
```

Comment:
Instead of manipulating browser history through a button click, we should wrap the `IconButton` in a `NavLink` that links to that path. This is better for accessibility & keyboard navigation, since it renders as an anchor tag. 

Something like 

```typescript
<NavLink to={`/app/clearing/member/${c.contractId.replace('#', '_')}`}>
   <IconButton color="primary" size="small" component="span">
       <KeyboardArrowRight fontSize="small" />
   </IconButton>
</NavLink>,
```

Reply:
(I'm doing the same thing all over the place in my WIP manage/setup screen PR 😅 )

Reply:
nice, will add it to the `Member.tsx` page too.

Reply:
Actually the ones on `Member.tsx` aren't links, they call handlers that exercise choices. Would it be better for those to have an `onClick` `NavLink` that links to `#`?

eg here: https://github.com/digital-asset/da-marketplace/blob/e87368d523b7f2aa6bd404eea9721a4cf8b5ab8b/ui2/src/pages/clearing/Member.tsx#L182

Reply:
yep, since it's not doing a `history.push`, those can be left as onClick buttons

Reply:
Also, possibly use the semantic ui `Button` with an icon classname instead of material. Since we are moving away from that..

Reply:
Good call - `Button` with the icon 

> Also, possibly use the semantic ui `Button` with an icon classname instead of material. Since we are moving away from that..

I switched these to use `ArrowRightIcon` from `icons.tsx`, and the icons on `Member.tsx` to be normal ol' buttons.